src/__fixtures__/users.js

```diff
@@ -9,8 +9,8 @@ const userAdminFixture = {
   github: 'TetsuoShima',
   teams: ['aabe63e7-87ef-4beb-a596-c810631fc474'], // platformTeamFixture
   scopes: [
-    new ObjectId('6751e606a171ebffac3cc9dd'), // breakGlassFixture
-    new ObjectId('7751e606a171ebffac3cc9dd') // adminScopeFixture
+    { scopeId: new ObjectId('6751e606a171ebffac3cc9dd') }, // breakGlassFixture
```

Comment:
do we actually need to use uuid's here? 
probably doesn't make a huge difference on the frontend etc, but if we ever need to dig into the data etc it might be easier to look at

Reply:
wanted to change as little as possible, for now, its done by id, but could move it to string, but then any rename of a permission would cause issues. same as teamIds I guess